src/utils.ts

```diff
@@ -70,7 +70,7 @@ export const duplicatedStoreWarningIf = (hasPersistedStoreAlready: boolean, stor
   }
 };
 
-export const computedPersistWarningIf = (isComputedProperty: boolean, propertyName: string): void => {
+export const computedPersistWarningIf = <P extends string>(isComputedProperty: boolean, propertyName: P): void => {
   if (isBrowser && isNotProductionBuild && isComputedProperty) {
     console.warn(`mobx-persist-store: The property '${propertyName}' is computed and will not persist.`);
   }
```

Comment:
@quarrant do you know why I get the following error:

```
$ tsc --noEmit
error TS2688: Cannot find type definition file for 'integration-test'.
  The file is in the program because:
    Entry point for implicit type library 'integration-test'

```

Reply:
Never mind it might of been something I did

Reply:
No idea....